source/includes/_fastlane.md

```diff
@@ -0,0 +1,240 @@
+| marketIndex       | The market index for the order                                                                                                                                                            | No       |              |
+| marketType        | The market type (must be set to "perp" for now)                                                                                                                                           | No       |              |
+| message           | Base64 string of Signed Message Hex Buffer (see above section)                                                                                                                            | No       |              |
+| signature         | Base64 string of order signature (see above section)                                                                                                                                      | No       |              |
+| taker_pubkey      | The authority wallet of the taker                                                                                                                                                         | No       |              |
+| signing_authority | The authority wallet that signed the order message (usually the same as taker_pubkey, different if signer is a delegate). If not provided, will assume taker_pubkey is the message signer | Yes      | taker_pubkey |
+
+```typescript
+const response = await axios.default.post(
+  "https://fastlane.drift.trade/orders",
+  {
+    market_index: 0,
+    market_type: "perp",
+    message: message.toString(),
+    signature: signature.toString("base64"),
+    taker_pubkey: driftClient.wallet.publicKey.toBase58(),
+  },
+  {
+    headers: {
+      "Content-Type": "application/json",
+    },
+  }
+);
+```
+
+## Makers: Subscribing to Orders Messages
+
+Drift sdks offer a FastlaneOrderSubscriber as a wrapper around the Fastlane websocket to easily subscribe to fastlane orders. Traders can also subscribe directly to the websocket feed and gain access to trades that way.
+
+To start receiving messages from the fastlane server, you must authenticate your connection with your wallet by signing a nonce sent by the server. This all happens under the hood in the FastlaneOrderSubscriber, but examples for how to do this for direct connections are given below
+
+### Subscribing to Websocket Server Directly
+
+WS Server URLs:
+
+mainnet-beta: [wss://fastlane.drift.trade/ws](wss://fastlane.drift.trade/ws)
+devnet: [wss://master.fastlane.drift.trade/ws](wss://master.fastlane.drift.trade/ws)
+
+#### Authenticating
+
+Once a connection has been established, the websocket server will send a message with a nonce. The message will take the shape of:
+
+{
+channel: 'auth',
+nonce: 'XXXX'
+}
+
+See the typescript example for how to decode the nonce message and sign and send back to the server.
+
+```typescript
+import nacl from 'tweetnacl';
+
```

Comment:
maybe its worth showing the Ws connect part too, since its not only the base url e..g "<base_url>/ws?pubkey=<KEY>"